tests/host_tests.py

```diff
@@ -22,35 +20,26 @@ def test_datadog_agent_is_installed(host):
     host.file("/usr/local/bin/datadog-agent").exists
 # Test if Timezone is set to UTC
 def test_timezone_is_set_to_utc(host):
-    output = host.run("sudo systemsetup -gettimezone").stdout
-    assert "GMT" in output
+    assert "GMT" in host.run("sudo systemsetup -gettimezone").stdout
 # Test if spotlight indexing is disabled
 def test_spotlight_indexing_is_disabled(host):
-    output = host.run("sudo mdutil -s /").stdout
-    assert "disabled" in output
+    assert "disabled" in host.run("sudo mdutil -s /").stdout
 # test if spotlight is disabled
 def test_spotlight_is_disabled(host):
-    output = host.run("sudo sudo mdutil -a -i off /").stdout
-    assert "disabled" in output
+    assert "disabled" in host.run("sudo mdutil -a -i off /").stdout
 # Test if admin SSH key is present
 def test_ssh_key_is_present(host):
     host.file("$HOME/.ssh/id_rsa.pub").exists
 # Test if tools are installed
 def test_anka_is_available(host):
-    output = host.run("/usr/local/bin/anka version").rc
-    assert output == 0
+    assert host.run("/usr/local/bin/anka version").rc == 0
 def test_packer_is_available(host):
-    output = host.run("/usr/local/bin/packer version").rc
-    assert output == 0
+    assert = host.run("/usr/local/bin/packer version").rc == 0
```

Comment:
`=` after assert is not needed (same comment to all the lines below this)

Reply:
you mean I should get rid of the `== 0` part because assert can evaluate the truthly value of the first argument?

Reply:
I mean the equal sign after assert `assert = host.run`
```suggestion
    assert host.run("/usr/local/bin/packer version").rc == 0
```

Reply:
@MrBlaise

Reply:
oooo I get it, those are copy pasta mistakes 🤦‍♂️